test/account-snapshot/snapshot.test.ts

```diff
@@ -0,0 +1,110 @@
+import {
```

Comment:
Nice! Lets keep this under /document-models/account-snapshot/test/ so it works almost as a monorepo.